app/models/sample.rb

```diff
@@ -747,7 +748,13 @@ def kickoff_pipeline
 
     pr.alignment_config = AlignmentConfig.find_by(name: alignment_config_name) if alignment_config_name
     pr.alignment_config ||= AlignmentConfig.find_by(name: AlignmentConfig::DEFAULT_NAME)
-    pr.save
+    pr.save!
+  rescue StandardError => err
+    LogUtil.log_err_and_airbrake("Error saving pipeline run: #{err.inspect}")
+    # This will cause a message to be shown to the user on the sample page.
+    # See app/assets/src/components/utils/sample.js
```

Comment:
nit: Ok, to leave it... but normally  Ido not comment on behavior of the client based on the backend behavior since client can change behavior later.

Reply:
That's a fair point. I will change to `This may cause a message`.

Reply:
As I think about it, this is not an upload error, right? We should find a better field and status for this.

Reply:
True, this is not an upload error in the sense that the files have been copied successfully. The user message is "issue processing". 

FWIW, we already overload `upload_error` with `DO_NOT_PROCESS`. 

I don't think it is worth creating a new field for this type of error and I'm not sure where else it would go. 

I think it's a big improvement over showing "waiting for files", the current user facing message.

Reply:
I guess at this point it is more the name of the field that is inaccurate. And is true that it will be unlikely for this to happen in staging and prod.
Can you add a comment with argument on why we use this field (e.g. the do not process precedent)?